pkg/plugins/platform/local.go

```diff
@@ -451,68 +464,62 @@ func (l *LocalApp) Config() error {
 	docroot := l.Docroot()
 	settingsFilePath := filepath.Join(basePath, docroot)
 
-	if l.GetType() == "drupal7" || l.GetType() == "drupal8" {
+	switch l.GetType() {
+	case "drupal8":
+		fallthrough
+	case "drupal7":
 		settingsFilePath = filepath.Join(settingsFilePath, "sites", "default", "settings.php")
```

Comment:
This still isn't great, I agree, but I do think this is an improvement. I'm hesitant to spend more time here right now, as this functionality is in a strange place I feel, and I'm not exactly sure what else to do with it right now. Its use is limited, and the configuration provided is the bare minimum to make a generic site work. 

We should probably give some more consideration to how deeply we want to support different CMS, and if we want to be handling platform-specific configurations. On one hand, it can make it easier to demonstrate a fully-working site, but on the other if we don't take these on, we can more easily support other platforms.